lib/desired-caps.js

```diff
@@ -168,6 +168,9 @@ let desiredCapConstraints = _.defaults({
   skipLogCapture: {
     isBoolean: true
   },
+  eventloopIdleDelaySec: {
```

Comment:
`wdaEventloopIdleDelay` here as README?

Reply:
Yes of course. Mixed the two names up...